### ISSUE TITLE
Forge 1.16.4 SMP Self-hosted server doesn't seem to detect DynMap v3.1-beta-7

### ISSUE THREAD

**Issue by 4xMeister** [webbukkit/dynmap#3276]
**Issue Description:** Copied DynMap jar into mods folder on Forge SMP Self-hosted Server. When started, nothing shows in the Server log window relating to DynMap. DynMap folder is never created in the mods folder. DynMap commands are not recognized as valid commands in either the Server command window or logged in as a player with 'op' permissions.
Forge Minecraft Server just doesn't seem to recognize the DynMap jar at all.
I have tried redownloading and copying over the DynMap jar with no luck.
I have removed all other mods in the mods folder, so DynMap is the Only mod in there.

* **Dynmap Version:** v3.1-beta-7
* **Server Version:** forge 1.16.4 build 35.1.4
* **Pastebin of Configuration.txt:** N/A - DynMap folder is never even created
* **Server Host (if applicable):** Selfhosted
* **Pastebin of crashlogs or other relevant logs:** nothing in logs - no crashes, just acts like DynMap isn't even in the mods folder
* **Other Relevant Data/Screenshots:** N/A
* **Steps to Replicate:** Install a clean install of 'forge-1.16.4-35.1.4-installer.jar', setup as usual-eula and all that good stuff, copy DynMap v3.1-beta-7 to mods folder and see that the forge server acts like it never even sees it in the mods folder - no DynMap folder created, no DynMap commands are available, nothing shows in the Server Command window on Server Startup - simply nothing at all.
 [I can't find anyone else mentioning this issue] *I have looked at all other issues and this is not a duplicate*  
 [Happens every time] *I have been able to replicate this*

**Comment by l1m3r** [comment]
uhm, you're using `forge-1.16.4-35.1.4-installer.jar` but what's the exact filename of the DynMap jar you're using?
From your report it sounds like you didn't use the **mod** `Dynmap-3.1-beta7-forge-1.16.5.jar` but maybe the **plugin** `Dynmap-3.1-beta7-spigot.jar`....

asking just to make sure this is not the problem.

**Comment by 4xMeister** [comment]
Hi, 
I am using the mod Dynmap-3.1-beta7-forge-1.16.4.jar
I also run a spigot minecraft server (completely separate machine) so I understand why using the plugin instead of the mod wouldn't work. :-)
Interestingly enough - I did do some more troubleshooting and see that if I startup minecraft forge-optifine 1.16.4 (the game, not the server [which are both installed into the same folder], if I go into the 'mods' option from inside the game, I see that  Dynmap-3.1-beta7-forge is indeed loaded in there. But of course, it doesn't make any Dynmap configuration folders in either the 'mods' folder, or even the 'config' folder (which I know is used for single player), but I checked just in case. ;-)
Am I doing something wrong?
Can I Not have Forge 1.16.4 server installed into the same folder as the single player game forge 1.16.4? It works fine for everything else, and seems to be the standard way to install forge 1.16.4 server for a local LAN implementation. But, I could be mistaken.
